### DISCUSSION TITLE
For the drawing orientation of AIS_ColorScale

### DISCUSSION THREAD

**Discussion by WY-Huang** [Open-Cascade-SAS/OCCT#150] [Q&A]
For the drawing orientation of AIS_ColorScale, the default is vertical. How can it be drawn horizontally?

**Comment by gkv311** [comment]
`AIS_ColorScale` doesn't provide an option for horizontal layout right now. You may play with mirror or rotation transformations, but I'm not sure it will be sufficient:
```
pload VISUALIZATION
vinit View1
vcolorscale vc -demo
vlocation vc -reset -mirror 0 0 0 -1 1 0
```

![IMG_20241115_150625_673](https://github.com/user-attachments/assets/c3aa1468-4f06-4bf8-9ca1-38e2c03806af)
